lib/HTTP/Status.pm

```diff
@@ -111,6 +111,8 @@ sub is_error        ($) { $_[0] >= 400 && $_[0] < 600; }
 sub is_client_error ($) { $_[0] >= 400 && $_[0] < 500; }
 sub is_server_error ($) { $_[0] >= 500 && $_[0] < 600; }
 
+sub status_code         { %StatusCode; }
```

Comment:
I think the earlier PR requested `status_codes` rather than `status_code` for the function name.

Reply:
Thanks. I renamed.